docs/nuxt/getting-started.md

```diff
@@ -78,6 +78,13 @@ This service account file contains sensitive information and should **not be com
 echo service-account.json >> .gitignore
 ```
 
+:::tip
```

Comment:
The env setup suggested also doesn't work when I created a project with firebase frameworks + firebase emulators:start then run on localhost:5000. 

I suspect it's because service-account is not automatically copied over to .firebase. Still haven't found a workaround for this.

![Pasted_Image_12_31_23__4_08_PM](https://github.com/vuejs/vuefire/assets/8771186/60c8e71f-ca0c-4295-b6aa-14b1ccd298ac)